locales/en/index.yml

```diff
@@ -273,6 +273,50 @@ authentication:
   expiredSessionBanner:
     title: Security notice
     message: To keep you private information safe, please login again with your SPID credentials, thanks!
+email:
+  read: 
+    title: Your email address
+    info: This is the email address associated to your SPID account. You can
+      use in IO app a different email address, if you wish. To make operative the
+      new email address, you will need to validate it by clicking on the link you
+      will receive by email.
+    details: You can change this adress, that you should validate clicking on 
+      the link we will sent to your email address. In the meanwhile, those 
+      functionalities of IO depending on the email (payments, messages forwarding,
+      etc.) will be disabled.
+    alert: "Warning: your PAgoPa account is associated to your email. 
+      So, if you change it, IO will no more list your registered payment methods
+      and transactions."
+    help: Your SPID profile communicates your email to IO app. If you find the address
+      wrong or you no longer use it, you can edit it directly from your SPID services
+      provider account.
+  insert:
+    header: Activation
+    title: Insert your email
+    subtitle: Some IO app functionalities need your email.
+    label: Personal email address
+    help:
+      title: Email
+      content: !include email_insert_help.md
+  edit:
+    title: Modify your email
+    subtitle: The email you inserted is
+    validated: The email you both inserted and validated is
+    label: New personal email address
+    cta: Edit email
+  validate:
+    title: Ricorda di validare il tuo indirizzo email
```

Comment:
It is in italian

Reply:
reverted to the previous value of strings within https://github.com/teamdigitale/io-app/pull/1363/commits/2d1442691356a14fc1b5864a0a30220da184f8cb